### PR TITLE
Add /check endpoint

### LINKED ISSUE #67: Add /check endpoint
This endpoint should be used to check if the server is alive and ready to receive requests

### PR DESCRIPTION
**Pull request by fmrsabino**
Closes #67 

- Adds `/check` endpoint that can be used to check the readiness for the service to receive requests

### CODE REVIEW

File: src/config/urls.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 from django.contrib import admin
-from django.urls import include, path
+from django.http import HttpResponse
+from django.urls import include, path, re_path
 
 urlpatterns = [
     path("api/v1/", include("safe_apps.urls", namespace="v1")),
     path("admin/", admin.site.urls),
+    re_path(r"^check/", lambda request: HttpResponse("Ok"), name="check"),
```
Comment (Uxio0) [inline src/config/urls.py:8]: Better use path instad of `re_path`, I should change it too :)

**Review by Uxio0 — CHANGES_REQUESTED** [review]

File: src/config/urls.py (hunk 1 above)
Comment (Uxio0) [reply src/config/urls.py]: Well, I use `re_path` everywhere, so I might leave it that way as it's not deprecated

Comment (fmrsabino) [reply src/config/urls.py]: Yeah I started with `path` but was wondering if I should change it to `re_path` as you have 🤔

Comment (fmrsabino) [reply src/config/urls.py]: Can `path` have issues with ending the endpoint with `/` or without it? I've tested `check/` and `check` and both worked fine with `path`

Comment (Uxio0) [reply src/config/urls.py]: There's an auto redirect, but stick with `check/`

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**